### DISCUSSION TITLE
New UI using Angular Cli v14

### DISCUSSION THREAD

**Discussion by wisebaldone** [Netflix/lemur#4196] [Ideas]
Would the project be open to a new UI to be developed?

**Comment by hosssha** [comment]
Hi @wisebaldone, we would certainly welcome a new UI, specially as the current version is way out of date.

**Reply by joeinfor7685** [reply]
@hosseinsh - hi, just noticed this discussion. we are working to deploy Lemur in our production environment and are currently running into a number of security vulnerabilities with Lemur running Flask 1.x and AngularJS 1.8.x.

We are open to contribute to update the Lemur UI. We would prefer developing the UI in React because we can leverage our current stack. If this is a deal breaker we can use the latest version of Angular, for sure.

Also, will the project be open to updating to Flask 2.0+? This will address the security vulnerabilities we are running into and bring the API up to date, as well.

Thanks!

**Reply by hosssha** [reply]
Hi @JoeMcRobot,

any contributions on the Flask migration and also upgrading the outdated Lemur UI is welcomed. We are happy to review the PRs.
I think React is the right future for the Lemur UI.
